h/data_tasks/report/refresh/01_annotations_refresh.sql

```diff
@@ -56,7 +56,11 @@ ON CONFLICT (uuid) DO UPDATE SET
     anchored=EXCLUDED.anchored,
     size=EXCLUDED.size,
     parent_uuids=EXCLUDED.parent_uuids,
-    tags=EXCLUDED.tags;
+    tags=EXCLUDED.tags,
+    imported=EXCLUDED.imported;
 
 -- Do we want to analyze every time we insert? This could take a while?
 ANALYSE report.annotations;
+
+-- Create the last added index here in `refresh` to avoid a full recreate
+CREATE INDEX CONCURRENTLY IF NOT EXISTS annotations_imported_idx ON report.annotations (imported);
```

Comment:
We'll try to create the index on `refresh`. It will create it the first time and the following times if will be just be there (and skipped because of the `IF NOT EXISTS`).